projProject/project.cpp

```diff
@@ -60,37 +210,76 @@ void filledInCircle(int x, int y, GLfloat radius) {
     glEnd();
 }
 
+//Draws frame for windows
+void drawCircle(float xC, float yC, int radius)
+{
+	const float pi = 3.141592653f;			// constant PI value
+	for (float angle = 0; angle <= 2 * pi; angle += 0.01)	{
+		glBegin(GL_POINTS);					// uses points to draw face
+			glVertex2f(xC + (radius*cos(angle)), yC + (radius*sin(angle)));				// draw the vertex point at the specified co-ordinate
+		glEnd();
+	}
+}
+
 // Draws 2 windows
 void drawWindows() {
-    glColor3f(.2, .4, 0);
-    filledInCircle(-100, -75, 20);
-    filledInCircle(100, -75, 20);
-}
+	//draws window frame
+	glPointSize(3);
+	glColor3f(0, 0, 0);
+	drawCircle(-100, -15, 50);
+	drawCircle(100, -15, 50);
 
-// Draws a house
-void drawHouse() {
-    glColor3f(0, .2, .4);
-    glBegin(GL_POLYGON);
-    glVertex2i(-200, -300);         // Bottom left
-    glVertex2i(-200, 0);           // Top left
-    glVertex2i(200, 0);            // Top right
-    glVertex2i(200, -300);          // Bottom right
-    glEnd();
+	//draws windows glass
+	glColor3f(0.439, 0.502, 0.565);		//slate grey
+	filledInCircle(-100, -15, 48);
+	filledInCircle(100, -15, 48);
+
+	glLineWidth(2.5);
+	// draws window pane
+	glColor3f(0, 0, 0); // color black
+	glBegin(GL_LINE_STRIP);
+	glVertex2i(-148,-15);
+	glVertex2i(-52, -15);
+	glEnd();
+	/*glBegin(GL_LINE_STRIP);
+	glVertex2i(-100, 33);
+	glVertex2i(-100, -63);
+	glEnd();*/
+
+	glBegin(GL_LINE_STRIP);
+	glVertex2i(52, -15);
+	glVertex2i(148, -15);
+	glEnd();
+	/*glBegin(GL_LINE_STRIP);
+	glVertex2i(100, 33);
+	glVertex2i(100, -63);
+	glEnd();*/
 }
 
+// Draws bird at x, y location
+void drawBird(int x, int y) {
+	glPixelStorei(GL_UNPACK_ALIGNMENT, 1);     // Only read one byte at a time
+	glColor3f(0, 0, 0);
+	glRasterPos2i(x, y);
+	glBitmap(16, 7, 0, 0, 0, 0, bird);
+}
 
 void myInit() {
 	glClearColor(1, 1, 1, 0);			// specify a background clor: white
 	gluOrtho2D(-300, 300, -300, 300);	// specify a viewing area
 }
 
-
 void myDisplayCallback() {
 	glClear(GL_COLOR_BUFFER_BIT);
+	drawBackground();
     drawHouse();
     drawWindows();
     drawDoor();
     drawHouseNumber();
+	drawBird(-200, 200);
```

Comment:
Why are these indented?

Reply:
I will add indentation before i merge it to master